examples/api/lib/material/material_state/material_state_mouse_cursor.0.dart

```diff
@@ -17,7 +17,10 @@ class MaterialStateMouseCursorExampleApp extends StatelessWidget {
       home: Scaffold(
         appBar: AppBar(title: const Text('MaterialStateMouseCursor Sample')),
         body: const Center(
-          child: MaterialStateMouseCursorExample(),
+          child: MaterialStateMouseCursorExample(
+            // Switch to get a different mouse cursor while hovering ListTile.
```

Comment:
```suggestion
            // TRY THIS: Switch to get a different mouse cursor while hovering ListTile.
```

(We use this form in other examples)

Reply:
@gspencergoog thanks for the suggestions!